ros/src/models/chrono/ros_chrono/src/path_following.cpp

```diff
@@ -115,12 +115,14 @@ double fps = 60;
 int filter_window_size = 20;
 
 // Control Input
-std::vector<double> traj_t(1,0);
-std::vector<double> traj_x(1,0);
-std::vector<double> traj_y(1,0);
-std::vector<double> traj_psi(1,0);
-std::vector<double> traj_sa(1,0);
-std::vector<double> traj_ux(1,0);
+std::vector<double> traj_t(2,0);
```

Comment:
why do you need to hard-code these parameters?? This should be a general function

Reply:
These variables actually don't need to be given an initial value, because the value will be overwritten in the callback function. So I deleted the initial values in the latest commit https://github.com/ChangYuHsuan/MAVs/pull/11/commits/d99b4e373c142b98b9fc4e4eb439f7c423a77285. I have built and tested that domoK still works successfully without any warning or error.